### PR TITLE
more testing 

### PR DESCRIPTION
**Pull request by akintner**
- add more tests for foods.js
This includes a test for localStorage persistence across page refreshes

### CODE REVIEW

**Comment by akintner** [comment]
@Dpalazzari 
@case-eee 
@icorson3 

More tests coming, this is an open PR WIP right now

**Comment by akintner** [comment]
- update to include tests for index.html

File: test/foods-test.js [hunk 1]
```diff
@@ -83,23 +95,52 @@ describe('#create-form', function() {
     });
   });
 
-  context('updating food name or calories', function(){
-    it('can change the name of a food when table row is clicked', function(){
+  context('editing food name or calories', function(){
+    xit('can change the name of a food when table row is clicked', function(){
       $('#name-field input').val('Pizza Slice');
```
Comment (Dpalazzari) [inline test/foods-test.js]: We will come back to this test if there is time.

File: test/index-test.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      $('.all-food-table').find('#food-row input').click();
+      prependCheckedFood('button#add-breakfast', breakfastTable, 400);
+    
+      assert.include(breakfastTable, '');
+    });
+  });
+
+  context('deleting foods from table', function() {
+    xit('with one food, it will delete the food row', function() {
+      $('#name-field input').val('Banana');
```
Comment (Dpalazzari) [inline test/index-test.js]: same here! 👍

File: test/index-test.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      var tableData = $("tbody").text();
+      assert.equal(tableData, foodData);
+
+      $('#delete-food').click();
+
+      var tableData = $("tbody").text();
+
+      assert.equal(tableData, "banana105-");
+    });
+  });
```
Comment (Dpalazzari) [inline test/index-test.js]: Mark it down for a return journey 👍

**Comment by akintner** [comment]
- test added for checking the date on the index page

- test added for checking methods in index.js

- test added but not working for adding checked foods to a meal table

**Comment by akintner** [comment]
@Dpalazzari  ready for merge

**Pull request merged**